README.md

```diff
@@ -45,6 +45,7 @@ Parsing and verifying tokens is pretty straight forward.  You pass in the token
 	token.Claims["foo"] = "bar"
 	token.Claims["exp"] = time.Now().Add(time.Hour * 72).Unix()
 	// Sign and get the complete encoded token as a string
+  // mySigningKey could for example be any random string
```

Comment:
* This is only true if you're using a symmetric signing method
* It's not a string, it's a `[]byte`
* 'any random string' is not that helpful as guidance